addons/website_slides/views/slide_channel_partner_views.xml

```diff
@@ -18,12 +19,15 @@
             <field name="model">slide.channel.partner</field>
             <field name="arch" type="xml">
                 <tree string="Attendees" editable="top">
+                    <field name="active" invisible="1"/>
                     <field name="create_date"/>
                     <field name="partner_id" string="Contact"/>
                     <field name="partner_email"/>
                     <field name="channel_id" string="Channel" invisible="context.get('default_channel_id',False)" />
                     <field name="completion" string="Progress" widget="progressbar" />
                     <button name="unlink" title="Remove" icon="fa-times" type="object"/>
+                    <button name="action_archive" title="Archive" icon="fa-archive" type="object" attrs="{'invisible': [('active', '=', False)]}"/>
+                    <button name="action_unarchive" title="Unarchive" icon="fa-archive" type="object" attrs="{'invisible': [('active', '=', True)]}"/>
```

Comment:
same icon for different action can be confusing.
Maybe we could use another icon for unarchive like 'undo' or else. Check with PO :)

Reply:
Indeed, I thought about it too but I didn't know what icon to use.
Undo is a good idea I think, what do you think @sbuhl ?